cmd/main.go

```diff
@@ -224,5 +226,7 @@ func Main() {
 	}
 
 	// Run the app - exit on error.
-	app.RunAndExitOnError()
+	if err := app.Run(args); err != nil {
+		os.Exit(1)
```

Comment:
You should take a callback for example for `cmd.Main(args []string, exitFunc func())`  - this way you can change the behavior by default it will be `os.Exit(1)` upon error. 

Otherwise for tests you can set a exit mock function.